cypress/fixtures/source_repos.json

```diff
@@ -1,5 +1,81 @@
 {
     "vela": [
+        {
+            "id": 23,
+            "user_id": 9,
+            "org": "vela",
+            "name": "api-2",
+            "full_name": "vela/api-2",
+            "link": "https://github.com/go-vela/api-2",
```

Comment:
Not a big deal but in all of the backend components we use `github/octocat` as the default test data.

Reply:
resolved, changed `vela` org to be `github` and made repos `octocat-...`